### DISCUSSION TITLE
Very Big question !

### DISCUSSION THREAD

**Discussion by blondie63** [codemagic-ci-cd/codemagic-docs#2048] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

It's very easy to add a new app in Codemagic, ok
I've created one for test and now i want to delete it
No delete button
Searching on docs, no result
Is this a strange question ?

### Expected results

NO

### Actual results

NO

**Comment by mikhail-tokarev** [comment]
> Is this a strange question ?

not at all :) go to the app settings > Repository settings > and you will see `Danger zone` section

**Reply by blondie63** [reply]
I don't want to delete my repo !!
I want to delete a codemagic app
<img width="1505" alt="Screenshot 2023-09-15 alle 17 55 57" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/1261851/1279eab8-8136-4a82-84a8-704c295bdf7c">

**Comment by mikhail-tokarev** [comment]
it won’t delete your repository just the app. On 15. Sep 2023, at 17:57, Mauro Miotello ***@***.***> wrote:﻿
I don't want to delete my repo !!
I want to delete a codemagic app

**Comment by blondie63** [comment]
On App/Settings/Danger Zone there's only the button to delete the Repo..

<img width="882" alt="Screenshot 2023-09-16 alle 08 26 37" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/1261851/8a6e8b2d-c812-467e-8426-f9a5e772e408">

**Comment by blondie63** [comment, marked as answer]
ok i've found it !
right path is Codemagic/App/Settings/Repository Settings
